__tests__/iterable_source.test.js

```diff
@@ -21,6 +21,7 @@ inputData.forEach((input, index) => {
   it(`${name} Tests: payload: ${index}`, async () => {
     try {
       const output = await transformer.process(input);
+      delete output.properties.email;
```

Comment:
Why are we deleting some property in the test.js file where we can remove it directly from output.json.

Reply:
`dataFields` contain `email`, we are removing it from `properties`(in output) to reduce redundancy(as `email` is already present in `traits`).
To do this I deleted `email` from `properties` in test.js file.
As suggested, I will perform this action in transform.js file rather than in test.js file.